components/atom/progressBar/src/LineDoubleProgressBar/index.js

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomLineDoubleProgressBarV2'
+const CLASS_INDICATOR = `${BASE_CLASS}-indicator`
+const CLASS_CONTAINER_BAR = `${BASE_CLASS}-container`
+const CLASS_BAR = `${BASE_CLASS}-bar`
+const CLASS_BAR_ANIMATED = `${CLASS_BAR}--animated`
+const CLASS_EXTRA_BAR = `${BASE_CLASS}-extraBar`
+const CLASS_EXTRA_BAR_ANIMATED = `${CLASS_EXTRA_BAR}--animated`
+
+const Indicator = props => {
+  const {indicatorBottom, mainBarPercentage, indicatorTotal} = props // eslint-disable-line react/prop-types
+  return (
+    <span
+      className={cx(
+        CLASS_INDICATOR,
+        `${CLASS_INDICATOR}--${indicatorBottom ? 'bottom' : 'top'}`
+      )}
+    >
+      <strong>{mainBarPercentage}</strong>
+      {indicatorTotal ? `/100` : `%`}
+    </span>
+  )
+}
+
+const LineDoubleProgressBar = props => {
+  const {
+    mainBarPercentage,
+    extraBarPercentage,
+    indicatorBottom,
+    isAnimatedOnChange,
+    hideIndicator
+  } = props
+
+  const width = `${mainBarPercentage}%`
+  const widthExtraBar = `${extraBarPercentage}%`
+  const styleBar = {width}
+  const styleExtraBar = {width: widthExtraBar}
+
+  return (
+    <div className={BASE_CLASS}>
+      {!hideIndicator && !indicatorBottom && <Indicator {...props} />}
+      <div className={CLASS_CONTAINER_BAR}>
+        <span
+          className={cx(CLASS_BAR, isAnimatedOnChange && CLASS_BAR_ANIMATED)}
+          style={styleBar}
+        />
+        <span
+          className={cx(
+            CLASS_EXTRA_BAR,
+            isAnimatedOnChange && CLASS_EXTRA_BAR_ANIMATED
+          )}
+          style={styleExtraBar}
+        />
+      </div>
+      {!hideIndicator && indicatorBottom && <Indicator {...props} />}
+    </div>
+  )
+}
+
+LineDoubleProgressBar.displayName = 'LineDoubleProgressBar'
+
+LineDoubleProgressBar.propTypes = {
+  /** Percentage value to be displayed in main bar as number and as bar width  */
+  mainBarPercentage: PropTypes.number,
+
+  /** If the indicator should be displayed with the pattern → {percentage}/100 ({percentage}% as default) */
+  indicatorTotal: PropTypes.bool,
+
+  /** If the bar "value" (width) should be displayed with animation */
+  isAnimatedOnChange: PropTypes.bool,
+
+  /** If the indicator should be placed below the bar */
+  indicatorBottom: PropTypes.bool,
```

Comment:
```suggestion
  isBottomIndicator: PropTypes.bool,
```

perhaps?

although I'd much prefer exporting `position` types i.e.

```
POSITIONING: {
> BOTTOM: 'bottom',
> TOP: 'top'
}
```

Reply:
I agree with you but I tried to keep the same props as the other types of progress bar as much as I can.

Reply:
If we are changing the old prop naming of the component, lets change the boolean prop to a valid enum values

Reply:
I didn't change the name, I kept the same name _indicatorBottom_ in order to has the same props name in the different types of progress bar.